synopsys-task/src/synopsys-task/model/srm.ts

```diff
@@ -0,0 +1,32 @@
+import { CoverityArbitrary } from "./coverity";
+import { BlackDuckArbitrary } from "./blackduck";
+
+export interface Srm {
+  srm: SrmData;
+  coverity?: coverityDetails;
```

Comment:
Can you rename coverityDetails to CoverityDetails to follow consistent naming conventions across all.